system_baseline/views/v0.py

```diff
@@ -131,9 +157,27 @@ def delete_baselines_by_ids(baseline_ids):
     return "OK"
 
 
+def _create_ordering(order_by, order_how, query):
+    """
+    helper method to set ordering on query
+    """
+    if order_by == "display_name":
+        if order_how == "ASC":
+            query = query.order_by(SystemBaseline.display_name.asc())
+        else:
+            query = query.order_by(SystemBaseline.display_name.desc())
+    elif order_by == "created_on":
+        if order_how == "DESC":
```

Comment:
Minor, but would be nice if this matched the previous if-block by starting with ASC

Reply:
missed this before, thx